app/src/main/java/com/eventersapp/marketplace/ui/dashboard/ProfileFragment.kt

```diff
@@ -10,32 +10,37 @@ import androidx.fragment.app.Fragment
 import androidx.lifecycle.Observer
 import androidx.lifecycle.ViewModelProvider
 import androidx.navigation.fragment.findNavController
-import com.eventersapp.consumer.R
-import com.eventersapp.consumer.databinding.FragmentProfileBinding
-import com.eventersapp.consumer.ui.viewmodel.ProfileViewModel
-import com.eventersapp.consumer.ui.viewmodelfactory.ProfileViewModelFactory
-import com.eventersapp.consumer.util.*
+import com.eventersapp.marketplace.R
+import com.eventersapp.marketplace.databinding.FragmentProfileBinding
+import com.eventersapp.marketplace.ui.viewmodel.ProfileViewModel
+import com.eventersapp.marketplace.ui.viewmodelfactory.ProfileViewModelFactory
+import com.eventersapp.marketplace.util.*
 import org.kodein.di.KodeinAware
 import org.kodein.di.android.x.closestKodein
 import org.kodein.di.generic.instance
 
-class ProfileFragment : Fragment(), KodeinAware {
+
+class ProfileFragment : Fragment(), KodeinAware, View.OnClickListener {
 
     companion object {
-        private const val ADDRESS = "https://goalseeker.purestake.io/algorand/testnet/account/"
+        private const val ADDRESS = "<Algorand-URL>"//Replace with Algorand address url
```

Comment:
Add this in Readme.md, to replace purestake account

Reply:
Done